xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/java/org/xwiki/livedata/script/LiveDataScriptService.java

```diff
@@ -101,10 +112,10 @@ public LiveData query(Map<String, Object> queryConfig)
     public LiveData query(String queryConfigJSON)
     {
         try {
-            return query(this.configHelper.createQuery(queryConfigJSON));
-        } catch (Exception e) {
-            this.logger.warn("Failed to create the live data query from JSON [{}]. Root cause is [{}].",
-                queryConfigJSON, ExceptionUtils.getRootCauseMessage(e));
+            return query(new ObjectMapper().readValue(queryConfigJSON, Map.class));
+        } catch (JsonProcessingException e) {
+            this.logger.warn("Failed to convert the live data query parameters to a Map. Root cause is [{}].",
```

Comment:
hmm it would make more sense to throw an exception rather than log something. Also it's more than a warning it's an error (which is why thrown an exception would be much nicer).